holoviews/plotting/bokeh/chart.py

```diff
@@ -794,10 +801,8 @@ def get_data(self, element, ranges, empty):
 
         # Iterate over stacks and groups and accumulate data
         data = defaultdict(list)
-        baselines = defaultdict(float)
+        baselines = defaultdict(lambda: {True: 0, False: 0})
```

Comment:
I find this a little weird and I don't think the code would be much more complicated if you use:

```
defaultdict(lambda: {'positive': 0, 'negative': 0})
```

or

```
defaultdict(lambda: {'+ve': 0, '-ve': 0})
```